code/processes/rdb.q

```diff
@@ -9,6 +9,7 @@
 hdbtypes:@[value;`hdbtypes;`hdb];                           //list of hdb types to look for and call in hdb reload
 hdbnames:@[value;`hdbnames;()];                             //list of hdb names to search for and call in hdb reload
 tickerplanttypes:@[value;`tickerplanttypes;`tickerplant];   //list of tickerplant types to try and make a connection to
+requiredprocs:tickerplanttypes;                             //required processes 
```

Comment:
You probably don't need to make this a separate variable - just use tickerplanttypes

Reply:
I have put this here to be consistent with the rest of the code